mail.md

```diff
@@ -0,0 +1,35 @@
+**For the first time ever, Exercism is hiring!**
+Thanks to the generosity of our community and supporting organisations, we're about to take Exercism to a whole new level.
+ 
+---
+ 
+Hi exercism-bot,
+
+Since we started, we've been longing to get to a point where we can hire a full-time team to unleash Exercism's potential. 
+After the launch and success of the new platform at the end of last year, and generous support from our community and supporting organisations, we're finally there. 
+We couldn't be more excited! 🥳
+
+In hiring the team and growing Exercism, we have three main goals:
+
+- Firstly, we want to do more of what we've got right already - providing a fun way for developers to improve their skills and learn new languages. 
+  We have a whole load of new content planned, expanding from programming languages to wider programming expertise. 
+  Expect deeper content on programming concepts and practical skills such as refactoring and application design.
+- Secondly, we're going to be building out a full "Learn to Code" part of the platform aimed at total beginners designed in the hands-on Exercism way. 
```

Comment:
The "secondly" is a new bullet point.

Reply:
It is for sure, so is the next one.  It is not special except that it is the second one.  But we can see that, it may not need to be pointed out.  (And if moved or another bullet point inserted, this would also need to be changed.  Still, it probably is fine.